tabs-tabs-tabs/manifest.json

```diff
@@ -11,5 +11,13 @@
   "permissions": [
     "tabs"
   ],
-  "version": "1.0"
+  "version": "1.4",
+  "applications": {
+      "gecko": {
+          "id": "a@b.c"
+      }
+  },
+  "background": {
+      "scripts": ["background.js"]
+  }
```

Comment:
Is the change to this file intentional? It doesn't look related and there's not description of it.

Reply:
Totally accident. Will remove.